libxml/Makefile

```diff
@@ -0,0 +1,37 @@
+PYODIDE_ROOT=$(abspath ..)
+include ../Makefile.envs
+
+LIBXMLVERSION=2.9.9
+
+ROOT=$(abspath .)
+
+SRC=$(ROOT)/libxml2-$(LIBXMLVERSION)
+TARBALL=$(ROOT)/downloads/libxml-$(LIBXMLVERSION).tgz
+URL=ftp://xmlsoft.org/libxml2/libxml2-$(LIBXMLVERSION).tar.gz
+
+
+all: $(SRC)/.libs/libxml2.a
+
+
+clean:
+	-rm -fr downloads
+	-rm -fr $(SRC)
+
+
+$(TARBALL):
+	[ -d $(ROOT)/downloads ] || mkdir $(ROOT)/downloads
+	wget -q -O $@ $(URL)
+#	md5sum --quiet --check checksums || (rm $@; false)
```

Comment:
We should definitely check the checksums for security reasons.

Reply:
Cool, added sha256 checksums